app/models/group.rb

```diff
@@ -2,8 +2,8 @@ class Group < ActiveRecord::Base
   has_many :users
   has_many :requests
 
-  validates :primary_number, :street_name, :street_suffix, :city_name,
-            :state_abbreviation, :zipcode, presence: true
+  validates :primary_number, :street_name, :city_name, :state_abbreviation,
```

Comment:
@kimchanyoung Yes it can. Remember that this validation is only going to happen after the API gives us a normalized list of parameters. The reason we have to make the street_suffix parameter `optional` is because the API won't always return a street suffix. 

When Steven C. tested our app, the `street_name` came back as "Avenue of the Americas", but there was no `street_suffix`. He couldn't be assigned a group due to our validation.